contracts/vaults/Base.sol

```diff
@@ -383,22 +385,22 @@ contract  Base is Governable, ReentrancyGuard {
 
     function validateSplit(BountySplit memory _bountySplit) internal pure {
         require(_bountySplit.hackerVested
-            + _bountySplit.hacker
-            + _bountySplit.committee
-            + _bountySplit.swapAndBurn
-            + _bountySplit.governanceHat
-            + _bountySplit.hackerHat == HUNDRED_PERCENT,
-        "HVE29");
+        + _bountySplit.hacker
```

Comment:
this is not an improvement

Reply:
i think it's good that it's clear, but not necessary, can remove

Reply:
the linter should decide this anyway

Reply:
on variable names?..

Reply:
oh you mean the identation, not changing `hackerHat` to `hackerHatVested`?

Reply:
yess